README.md

```diff
@@ -412,8 +417,12 @@ timestamp.source=wallclock
 
 Integration tests are implemented using JUnit, Gradle and Docker.
 
-To run them, you need:
-- Docker installed.
+To run them, you need Docker. Instruction available [here](https://docs.docker.com/engine/install/fedora/)
+
+`On Error "docker: Got permission denied while trying to connect to the Docker daemon"`:
+```
+sudo chmod 640 /var/run/docker.sock
+```
```

Comment:
Not sure if we need to add this in this README

Reply:
Ok, I will remove this one.